test/e2e/antreaipam_service_test.go

```diff
@@ -0,0 +1,58 @@
+package e2e
+
+import (
+	"testing"
+)
+
+func TestAntreaIPAMClusterIPv4(t *testing.T) {
+	skipIfNotAntreaIPAMTest(t)
+	skipIfNotIPv4Cluster(t)
+
+	data, err := setupTest(t)
+	if err != nil {
+		t.Fatalf("Error when setting up test: %v", err)
+	}
+	defer teardownTest(t, data)
+
+	// Create AntreaIPAM IPPool and test Namespace
+	ippool, err := createIPPool(t, data, 0)
+	if err != nil {
+		t.Fatalf("Creating IPPool failed, err=%+v", err)
+	}
+	defer deleteIPPoolWrapper(t, data, ippool.Name)
+	annotations := map[string]string{}
+	annotations[IPPoolAnnotationKey] = ippool.Name
+	err = data.createNamespaceWithAnnotations(testAntreaIPAMNamespace, annotations)
+	if err != nil {
+		t.Fatalf("Creating AntreaIPAM Namespace failed, err=%+v", err)
+	}
+	defer deleteAntreaIPAMNamespace(t, data)
+
+	data.testClusterIP(t, false, testAntreaIPAMNamespace)
+}
+
+func TestAntreaIPAMNodePort(t *testing.T) {
+	skipIfNotAntreaIPAMTest(t)
+
```

Comment:
why don't it need to skipIfNotIPv4Cluster like above?
Actually the two tests are very similar, with much code can be shared.

Reply:
Thanks. Refactored to 1 test case and 2 sub test cases.

Reply:
skipIfNotIPv4Cluster is meaningless now since AntreaIPAM requires IPv4 currently.
AntreaIPAM e2e cases are variation of existing e2e cases, thus I tried to keep the original requirements.